# This service sets a list of rectangular prisms to be ignored 
# (corresponding to the fingers) 

int64 num_regions
float64[] widths	# Array of length 3*N, stores the widths of the prisms along x, y, and z directions
float64[] trans		# Array of length 3*N, stores the translation of each region
float64[] quats		# Array of length 4*N, stores the rotation of each region

---

int64 ret	# Whether we were successfully able to change bounds
string msg 	# Any error message associated with it
